src/v2/view-builder/components/AuthenticatorOptions.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+import { ListView, loc } from 'okta';
+import skipAll from './views/SkipOptionalEnrollmentButton';
+import FactorRow from './views/FactorRow';
+
+export default ListView.extend({
+
+  className: 'enroll-factor-list',
+
+  item: FactorRow,
+
+  itemSelector: '.list-content',
+
+  initialize: function () {
+    this.listenTo(this.collection,'selectFactor', function (data) {
+      this.model.set(this.options.name, data);
+      this.options.appState.trigger('saveForm', this.model);
+    });
+    this.hasOptionalFactors = this.options.appState.hasRemediationObject('skip');
+    if (this.hasOptionalFactors) {
+      this.model.set('formName', 'skip');
```

Comment:
Another thought: the skip logic shall go to View instead of in this component, which is only intent for rendering factor list.
The problem is we're reusing `SelectFactorEnrollView.js` for both factor and authenticator.
I have no problem to copy `SelectFactorEnrollView.js` into `SelectFactorAuthenticatorView.js` (same for `SelectFactorAuthenticateView.js`) and leave former untouched and change later to render skip button. Two ways to implement skip button
- change save button name to skip and override BaseForm.saveForm
- remove Save button and add a customized Skip button (see how Magih did for webauthn case: https://github.com/okta/okta-signin-widget/pull/1215)

With the copying, we would leave Factor related view completely untouched.